Dockerfile

```diff
@@ -68,38 +76,38 @@ RUN set -x \
     && grep " node-v$NODE_VERSION-linux-x64.tar.gz\$" SHASUMS256.txt.asc | sha256sum -c - \
     && tar -xzf "node-v$NODE_VERSION-linux-x64.tar.gz" -C /usr/local --strip-components=1 \
     && rm -r "$GNUPGHOME" "node-v$NODE_VERSION-linux-x64.tar.gz" SHASUMS256.txt.asc \
-    && apt-get purge -y --auto-remove wget
-
-RUN npm install -g yarn
+    && apt-get purge -y --auto-remove wget \
+    && npm install -g yarn@$YARN_VERSION \
+    && npm cache clear --force
 
-COPY yarn.lock /usr/src/app/
-COPY package.json /usr/src/app/
-RUN yarn install
+COPY package.json yarn.lock /usr/src/zeus/
+RUN yarn install --production --pure-lockfile --ignore-optional \
+    && yarn cache clean
 
-COPY requirements-base.txt /usr/src/app/
+COPY requirements-base.txt /usr/src/zeus/
 RUN pip install -r requirements-base.txt
 
-COPY requirements-dev.txt /usr/src/app/
+COPY requirements-dev.txt /usr/src/zeus/
 RUN pip install -r requirements-dev.txt
 
-COPY requirements-test.txt /usr/src/app/
+COPY requirements-test.txt /usr/src/zeus/
 RUN pip install -r requirements-test.txt
 
-COPY . /usr/src/app
-RUN pip install -e .
-RUN node_modules/.bin/webpack --config=config/webpack.config.dev.js
+COPY . /usr/src/zeus
+RUN pip install -e . \
+    && NODE_ENV=production node_modules/.bin/webpack --config=config/webpack.config.dev.js
```

Comment:
Well its saying the environment is prod for all of Node, which is what I was implying. It might be true that only webpack is using it right now, but not guaranteed.